### PR TITLE
added future versions for chrome and firefox up to 99 and 96

### LINKED ISSUE #2475: updating chrome and firefox versions
I'm creating a pull request to update chrome versions up to 99 and firefox up to 96 similar to this one https://github.com/browscap/browscap/pull/2374/files . Creating an issue to get an issue number for the test.

### PR DESCRIPTION
**Pull request by ocluf**
updated version numbers similar to this pull request https://github.com/browscap/browscap/pull/2374/files

### CODE REVIEW

**Review by mimmi20 — APPROVED** [review]

File: tests/issues/issue-2475.php [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<?php
+
+return [
+    'issue-2370-A' => [
```
Comment (mimmi20) [inline tests/issues/issue-2475.php]: Please rename this test to "issue-2475-A"

File: tests/issues/issue-2475.php [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            'Device_Brand_Name' => 'unknown',
+            'RenderingEngine_Name' => 'Blink',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'Google Inc',
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true,
+    ],
+    'issue-2370-B' => [
```
Comment (mimmi20) [inline tests/issues/issue-2475.php]: Please rename this test to "issue-2475-B"

File: tests/issues/issue-2475.php [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            'Device_Brand_Name' => 'Apple',
+            'RenderingEngine_Name' => 'WebKit',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'Apple Inc',
+        ],
+        'lite' => true,
+        'standard' => false,
+        'full' => true,
+    ],
+    'issue-2370-A (lite)' => [
```
Comment (mimmi20) [inline tests/issues/issue-2475.php]: Please rename this test to "issue-2475-A (lite)" and move this test above "issue-2475-B"

**Review by mimmi20 — CHANGES_REQUESTED** [review]

File: tests/issues/issue-2475.php (hunk 1 above)
Comment (ocluf) [reply tests/issues/issue-2475.php]: done

File: tests/issues/issue-2475.php (hunk 2 above)
Comment (ocluf) [reply tests/issues/issue-2475.php]: done

File: tests/issues/issue-2475.php (hunk 3 above)
Comment (ocluf) [reply tests/issues/issue-2475.php]: done

**Review by mimmi20 — APPROVED** [review]

**Review by asgrim — APPROVED** [review]

**Pull request merged**